.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Test
+on: push
+
```

Comment:
@ericherman we could consider also triggering the test on a pull request (on a certain branch):  
```
on: push
      pull_request:
          branches:
             - master
```